_build/buildPDF.sh

```diff
@@ -18,7 +18,11 @@ if ! command -v wkhtmltopdf >/dev/null 2>&1; then
 fi
 	
 # run the PDF build with wkhtmltopdf
-wkhtmltopdf --page-size Letter --margin-top 0.75in --margin-right 0.75in --margin-bottom 0.75in --margin-left 0.75in --encoding UTF-8 $1 $2
+if [ -z "$3" ] ; then
+	wkhtmltopdf --page-size Letter --margin-top 0.75in --margin-right 0.75in --margin-bottom 0.75in --margin-left 0.75in --encoding UTF-8 $1 $2
+else
+	wkhtmltopdf --user-style-sheet $3 --page-size Letter --margin-top 0.75in --margin-right 0.75in --margin-bottom 0.75in --margin-left 0.75in --encoding UTF-8 $1 $2
```

Comment:
could revert from `if/then` to a single line using `${parameter:+word}`

    wkhtmltopdf ${3:+--user-style-sheet $3} --page-size Letter ...

(i can never remember this bash magic, i always do `man bash` then search for `##` to find parameter expansion...)